mica-webapp/src/main/conf/application-dev.yml

```diff
@@ -17,6 +17,9 @@ agate:
         name: mica
         key: changeit
 
+portal:
```

Comment:
Have this setting in the MicaConfig object instead. There is already a "Public URL" field; add a new one called "Portal URL"